serialization/src/main/scala/sbt/serialization/SerializedValue.scala

```diff
@@ -113,6 +131,10 @@ private final case class LazyValue[V](value: V, pickler: SPickler[V]) extends Se
   override def parse[T](implicit unpickler: Unpickler[T]): Try[T] =
     toJson.parse[T]
```

Comment:
yeah, maybe if `pickler.tag == unpickler.tag` just return the value, otherwise do the toJson.parse[T]